iOS/Flows/Conversation/Cells/MessagesTimeMachineCollectionViewLayout.swift

```diff
@@ -192,12 +193,6 @@ class MessagesTimeMachineCollectionViewLayout: TimeMachineCollectionViewLayout {
     private var initialZPosition: CGFloat = 0
     /// Items that that were visible before the animation started.
     private var indexPathsVisibleBeforeAnimation: Set<IndexPath> = []
-    
-    override func prepare(forAnimatedBoundsChange oldBounds: CGRect) {
-        super.prepare(forAnimatedBoundsChange: oldBounds)
-        
-        self.shouldScrollToEnd = true
-    }
```

Comment:
We shouldn't always scroll to the latest message when changing layouts. Instead we only do it when entering write mode.